packages/react-app/src/components/NetworkSelector.jsx

```diff
@@ -1,86 +1,74 @@
-import { Flex, Text } from '@chakra-ui/core';
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  Text,
+} from '@chakra-ui/core';
 import React, { useContext, useEffect, useState } from 'react';
-import Select, { components } from 'react-select';
 
 import { BridgeContext } from '../contexts/BridgeContext';
 import { Web3Context } from '../contexts/Web3Context';
 import { DownArrowIcon } from '../icons/DownArrowIcon';
 import { networkOptions } from '../lib/constants';
 
-const { Option, DropdownIndicator } = components;
+const SelectOption = props => {
+  const { onChange, network } = props;
+  const { icon, label } = network;
 
-const CustomSelectOption = props => {
-  const {
-    data: { icon, label },
-  } = props;
   return (
-    <Option {...props}>
-      <Flex
-        align="center"
-        cursor="pointer"
-        color="grey"
-        transition="0.25s"
-        _hover={{ color: 'blue.500' }}
+    <Flex color="grey" transition="0.25s">
+      <Button
+        background="transparent"
+        width="100%"
+        justifyContent="flex-start"
+        _hover={{ color: 'blue.500', background: 'gray.100', zIndex: 4 }}
+        onClick={() => onChange(network)}
       >
         {icon}
         <Text color="black" ml={2}>
           {label}
         </Text>
-      </Flex>
-    </Option>
+      </Button>
```

Comment:
I made these buttons because it seemed to [hook into the Popover `focusable`](https://chakra-ui.com/popover#basic-usage) event vs other elements tried. This allows keyboard actions to work with the popover and its options.